GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -1,14 +1,64 @@
 /**
- * This DebuggerClient connects to a websocket server and can dump
+ * An client side implementation of the Debugger
+ * @interface
+ */
+gdjs.DebuggerClient = function() {} // Necessary for Interface definition 
+
+/**
+ * Update a value of the {@link RuntimeGame} instance.
+ * @function
+ * @name gdjs.DebuggerClient#set
+ * @param {Array<string>} path - The path to modify.
+ * @param {any} newValue - The new Value.
+ */
+
+/**
+ * Call a method of the {@link RuntimeGame} instance.
+ * @function
+ * @name gdjs.DebuggerClient#call
+ * @param {Array<string>} path - The path to the method.
+ * @param {Array<any>} args - The arguments to pass the method.
+ */
+
+/**
+ * Dump all the relevant data from the {@link RuntimeGame} instance and send it to the server.
+ * @function
+ * @name gdjs.DebuggerClient#sendRuntimeGameDump
+ */
+
+/**
+ * Start profiling.
+ * @function
+ * @name gdjs.DebuggerClient#sendProfilerStarted
+ */
+
+/**
+ * Stop profiling.
+ * @function
+ * @name gdjs.DebuggerClient#sendProfilerStopped
+ */
+
+/**
+ * Send profiling results.
+ * @function
+ * @param {number} framesAverageMeasures The average FPS.
+ * @param {?} stats
```

Comment:
Let's keep `any` - not super useful and it would needing typing in at the profiler level.